packages/cli/src/databases/entities/ProjectRelation.ts

```diff
@@ -3,10 +3,11 @@ import { User } from './User';
 import { WithTimestamps } from './AbstractEntity';
 import { Project } from './Project';
 
+// personalOwner is only used for personal projects
 export type ProjectRole =
+	| 'project:personalOwner'
```

Comment:
This is using a role to define a restriction that's not related to roles. IMO, there should only be a `project:owner`, and the check for is the project is personal or not should reside elsewhere.

Reply:
So, this role will only ever be used for personal projects, hence the name. All others will be using the `project:admin` role. The main distinction will be in that `project:personalOwner` will be missing a couple of permissions (`project:update` and `project:invite`) since they won't work on personal projects.